waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -38,13 +42,23 @@ import Wasp.Generator.DbGenerator.Common
 import qualified Wasp.Generator.DbGenerator.Jobs as DbJobs
 import Wasp.Generator.FileDraft.WriteableMonad (WriteableMonad (copyDirectoryRecursive))
 import qualified Wasp.Generator.Job as J
-import Wasp.Generator.Job.IO (printJobMsgsUntilExitReceived, readJobMessagesAndPrintThemPrefixed)
+import Wasp.Generator.Job.IO
+  ( collectJobTextOutputUntilExitReceived,
+    printJobMsgsUntilExitReceived,
+    readJobMessagesAndPrintThemPrefixed,
+  )
 import qualified Wasp.Generator.WriteFileDrafts as Generator.WriteFileDrafts
 import Wasp.Project.Db.Migrations (DbMigrationsDir)
 import Wasp.Util (checksumFromFilePath, hexToString)
 import Wasp.Util.IO (deleteFileIfExists, doesFileExist)
 import qualified Wasp.Util.IO as IOUtil
 
+data DbConnectionTestResult
+  = DbConnectionSuccess
+  | DbNotCreated
+  | DbConnectionFailure
```

Comment:
I would be tempted to add `String` to `DbConnectionFailure` that contains actual error message, is there a reason why you decided to drop that information?

Reply:
Because we don't get an error message per se, but we get some command output that might be useless unless parsed.